libs/core/src/components/pds-popover/pds-popover.tsx

```diff
@@ -0,0 +1,220 @@
+      event.stopPropagation();
+      return;
+    }
+
+    if (event.key === 'Escape' && this.active) {
+      this.hide();
+      event.preventDefault();
+      event.stopPropagation();
+    }
+  }
+
+  @Listen('click', {
+    capture: true
+  })
+  handleClick(event: MouseEvent) {
+
+    if (event.composedPath()[0] !== this.el.shadowRoot.querySelector('.pds-popover__trigger')) {
+      event.stopPropagation();
+      return;
+    }
+
+    if (!this.active) {
+      this.show();
+    }
+    event.stopPropagation();
+  }
+
+  @Listen('click', {
+    target: 'document'
+  })
+  handleDocumentClick(event: MouseEvent) {
+    if (this.active && this.popoverType !== 'manual') {
+      this.hide();
+      event.stopPropagation();
+    }
+  }
+
+  @Listen('scroll', {
+    target: 'window',
+    capture: true
+  })
+  handleScroll() {
+    if (this.active) {
+      this.handlePopoverPositioning();
+    }
+  }
+
+  private handlePopoverPositioning() {
+    const triggerEl = this.el.shadowRoot.querySelector('.pds-popover__trigger') as HTMLElement;
+    const popoverEl = this.el.shadowRoot.querySelector('div[popover]') as HTMLElement;
+
+    if (!triggerEl || !popoverEl) return;
+
+    const triggerRect = triggerEl.getBoundingClientRect();
+    const popoverRect = popoverEl.getBoundingClientRect();
+
+    let top = 0;
+    let left = 0;
+    // right and bottom are working, but left and top are overlapping the trigger
+
+    switch (this.placement) {
+      case 'top':
+        top = triggerRect.top - popoverRect.height;
+        left = triggerRect.left + (triggerRect.width - popoverRect.width) / 2;
+        break;
+      case 'right':
+        top = triggerRect.top + (triggerRect.height - popoverRect.height) / 2;
+        left = triggerRect.right;
+        break;
+      case 'bottom':
+        top = triggerRect.bottom;
+        left = triggerRect.left + (triggerRect.width - popoverRect.width) / 2;
+        break;
+      case 'left':
+        top = triggerRect.top + (triggerRect.height - popoverRect.height) / 2;
+        left = triggerRect.left - popoverRect.width - triggerRect.width;
+        break;
+    }
+
+    popoverEl.style.top = `${top}px`;
+    popoverEl.style.left = `${left}px`;
+  }
+
+  /**
+   * Shows the popover by enabling the active state
+   */
+  @Method()
+  async show() {
+    this.active = true;
+    this.handlePopoverPositioning();
+
+    const popoverElement = this.el.shadowRoot.querySelector('[popover]') as HTMLElement;
+    if (Build.isBrowser) {
+      popoverElement.showPopover();
+    }
+
+    this.showPdsPopover.emit();
+  }
+
+  /**
+   * Hides the popover by disabling the active state
+   */
+  @Method()
+  async hide() {
+    this.active = false;
+    const popoverElement = this.el.shadowRoot.querySelector('[popover]') as HTMLElement;
+
+    if (Build.isBrowser) {
+      popoverElement.hidePopover();
+    }
+
+    this.hidePdsPopover.emit();
+  }
+
+  render() {
+    return (
+      <Host>
+        <button
```

Comment:
The trigger seems to only show the popover vs toggling. Meaning I can open it with the trigger, but clicking the trigger again does not close it. This may be intended so just let me know.

Reply:
Based on the Popover API, the trigger has 3 available props: `show`, `toggle`, and `hide`. Currently, the shadow is configured to `show`, but not `toggle`. Adding `toggle` into the mix allows that expected behavior, but I intentionally left it off because `show` is the default trigger behavior. Thoughts?

Reply:
That's super helpful to know and now I am not sure. It felt odd to me that I couldn't toggle it and had to explicitly move my mouse away and then click, but maybe that's just me. Maybe it's something we can bring up with the group if it pops up in the future.

Reply:
Added